frame/bounties/src/tests.rs

```diff
@@ -126,15 +124,14 @@ impl pallet_treasury::Config for Test {
 	type BurnDestination = (); // Just gets burned.
 	type WeightInfo = ();
 	type SpendFunds = Bounties;
-	type MaxApprovals = MaxApprovals;
+	type MaxApprovals = frame_support::traits::ConstU32<100>;
 }
 parameter_types! {
 	pub const BountyDepositBase: u64 = 80;
 	pub const BountyDepositPayoutDelay: u64 = 3;
 	pub const BountyUpdatePeriod: u32 = 20;
 	pub const BountyCuratorDeposit: Permill = Permill::from_percent(50);
 	pub const BountyValueMinimum: u64 = 1;
```

Comment:
how about the type above (except BountyCuratorDeposit).

Reply:
Alright